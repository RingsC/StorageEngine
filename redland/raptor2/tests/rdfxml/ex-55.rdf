<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE rdf:RDF [
   <!ENTITY foo 'abc'>
   <!ENTITY bar 'def'>
]>
<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
  <rdf:Description rdf:about="http://example.org/resource"
        rdf:value="&foo;">
     <rdf:value>&bar;</rdf:value>
  </rdf:Description>
</rdf:RDF>
