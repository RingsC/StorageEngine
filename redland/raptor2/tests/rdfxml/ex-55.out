<http://example.org/resource> <http://www.w3.org/1999/02/22-rdf-syntax-ns#value> "abc" .
<http://example.org/resource> <http://www.w3.org/1999/02/22-rdf-syntax-ns#value> "def" .
